docs/docs/cli/introduction.md

```diff
@@ -16,12 +16,17 @@ Airy Core instances directly from your terminal.
 
 </TLDR>
 
+import Script from "@site/src/components/Script";
+
+<Script data-cols="120" data-rows="32" id="asciicast-403849" src="https://asciinema.org/a/403849.js"></Script>
+
 Installing the Airy CLI is easy and straightforward. You can follow the next
 steps for a quick setup:
 
-- [Step 1: Check the requirements](installation.md#step-1-check-the-requirements)
-- [Step 2: Install the Airy CLI](installation.md#step-2-install-the-airy-cli)
-- [Step 3: Verify your installation](installation.md#step-3-verify-your-installation)
+- [Step 1: Check the requirements](introduction.md#step-1-check-the-requirements)
+- [Step 2: Install the Airy CLI](introduction.md#step-2-install-the-airy-cli)
+- [Step 3: Verify your installation](introduction.md#step-3-verify-your-installation)
+- [Step 4: Initialize the setup](introduction.md#step-4-initialize-the-setup)
```

Comment:
Why is this being added? I rebased on top of develop

Reply:
I think it was generated from the markdown plugin. Looks correct anyhow